### PR TITLE
Updating Robo Function run:test

### PR DESCRIPTION
**Pull request by puneet0191**
This PR would allow us to choose specific method that we would want to run in a cest file.

To test the PR:
```
vendor/bin/robo run:test
```
After you select the test name you would like to execute, Robo would ask to choose a specific method 
See image
image

### CODE REVIEW

**Comment by therunnergit** [comment]
In the `runTest` method this command is missing:

    $this->createTestingSite($use_htaccess);

This means that if the test site is not present, the test execution will fail. Is that intended?

**Comment by puneet0191** [comment]
Yeah I understand your point, we can add that command, but is a overhead in some sense because we will execute run:test many times, the assumption is once we have the testing site setup and created, we are running individual test, or while developing a new test we use this command and I am not sure if we should check if Joomla-cms exist everytime.

**Comment by therunnergit** [comment]
Fine for me.

File: RoboFile.php [hunk 1]
```diff
@@ -155,12 +155,70 @@ public function runTest($pathToTestFile = null, $suite = 'acceptance')
...
+					{
+						$this->say('[' . $i . '] ' . $method_name);
+						$methods[$i] = $method_name;
+						$i++;
+					}
+				}
+			}
+			$this->say('');
+			$methodNumber = $this->ask('Please choose the method in the test that you would want to run...');
+			$method = $methods[$methodNumber];
```
Comment (javigomez) [inline RoboFile.php]: After the merge of this pull we can probably move all this logic to jorobo project (https://github.com/joomla-projects/jorobo).

**Comment by javigomez** [comment]
test. Ok

**Comment by javigomez** [comment]
closing for #161

**Comment by puneet0191** [comment]
@javigomez Should I close this one?

**Pull request closed without merging**